azure-pipelines.yml

```diff
@@ -23,6 +23,12 @@ steps:
     echo '##vso[task.prependpath]$(GOROOT)/bin'
   displayName: 'Set up the Go workspace'
 
+- script: |
+    go get github.com/gobuffalo/packr/packr
```

Comment:
I think the build is failing because this installs packr v1 but the build is using `packr2`. I tried this out on my machine, and installing `github.com/gobuffalo/packr/v2/packr2` seemed to work. You may want to double check that first (or try it out against the CI and cross your fingers). :grinning:

Reply:
Indeed, that was it.  Thank you!  😅